board/views.py

```diff
@@ -35,10 +133,13 @@ def __init__(self, game_id):
             self.img_url = get_img_url(res['cover']['image_id'])
         else:
             self.img_url = get_img_url(res['screenshots'][0]['image_id'])
-        self.release = datetime.datetime.fromtimestamp(res['release_dates'][0])
+        if 'release_dates' in res:
+            self.release = datetime.datetime.fromtimestamp(res['release_dates'][0])
+        else:
+            self.release = '--.--.--'
```

Comment:
You should assign `None` here and handle formatting in template since this is the correct separation of concerns.